kolibri/deployment/default/settings/base.py

```diff
@@ -339,6 +339,19 @@
 
 ENABLE_DATA_BOOTSTRAPPING = True
 
+# Session configuration
+
+SESSION_ENGINE = "django.contrib.sessions.backends.file"
+
+SESSION_FILE_PATH = os.path.join(conf.KOLIBRI_HOME, "sessions")
+
+if not os.path.exists(SESSION_FILE_PATH):
+    if not os.path.exists(conf.KOLIBRI_HOME):
+        raise RuntimeError("The KOLIBRI_HOME dir does not exist")
+    os.mkdir(SESSION_FILE_PATH)
+
+SESSION_COOKIE_NAME = "kolibri"
```

Comment:
We should probably name the cookie `sessionid` if we do decide to continue with this method because setting it to `kolibri` breaks tools which manually sent cookies (like Velox e.g.)

Reply:
Note that I added this because of this issue: https://github.com/learningequality/kolibri/issues/3921

Reply:
Got it, makes sense.